skycultures/russian_siberian/index.json

```diff
@@ -7,21 +7,21 @@
     {
       "id": "CON russian_siberian Elk",
       "lines": [[67301, 65378, 62956, 59774, 58001, 53910, 54061, 59774]],
-      "common_name": {"english": "Elk", "native": "Лось"}
+      "common_name": {"english": "Elk", "native": "Лось", "pronounce": "ɫ̪os̪ʲ"}
     },
     {
       "id": "CON russian_siberian Kic",
       "lines": [[26241, 26268, 26727, 26311, 25930]],
-      "common_name": {"english": "Threshers", "native": "Кичиги"}
+      "common_name": {"english": "Threshers", "native": "Кичиги", "pronounce": "k̟ʲɪˈt̠̪͡ʲɕ̪iɡ̟ʲɪ"}
     },
     {
       "id": "CON russian_siberian DNe",
       "lines": [[17499, 17489, 17531, 17579, 17664, 17851, 17847, 17608, 17499]],
-      "common_name": {"english": "Duck Nest", "native": "Утиное гнездо"}
+      "common_name": {"english": "Duck Nest", "native": "Утиное гнездо", "pronounce": "uˈt̪ʲin̪ə.jə ɡn̪ʲɪˈz̪d̪o"}
```

Comment:
Obvious fix: Utinoje gnezdo

Reply:
Not obvious at all. The "j" will be read differently by German, French, English, and Spanish people. And it definitely isn't [j] in English.

Reply:
Yeah, I’ve mixed styles, sorry. It should be: Utinoye gnyezdo